games-emulation/jollycv-jg/jollycv-jg-1.0.1.ebuild

```diff
@@ -0,0 +1,46 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit toolchain-funcs
+
+MY_PN=${PN%-*}
+MY_P=${MY_PN}-${PV}
+DESCRIPTION="Jolly Good ColecoVision Emulator"
+HOMEPAGE="https://gitlab.com/jgemu/jollycv"
+if [[ "${PV}" == *9999 ]] ; then
+	inherit git-r3
+	EGIT_REPO_URI="https://gitlab.com/jgemu/${MY_PN}.git"
+else
+	SRC_URI="https://gitlab.com/jgemu/${MY_PN}/-/archive/${PV}/${MY_P}.tar.bz2"
+	S="${WORKDIR}/${MY_P}"
+	KEYWORDS="~amd64"
+fi
+
+LICENSE="BSD MIT"
+SLOT="1"
+
+DEPEND="
+	media-libs/speexdsp
+"
+RDEPEND="
+	${DEPEND}
+	games-emulation/jgrf:1=
```

Comment:
it purely needs jgrf at runtime, not at build time?

If it's runtime only, I'd expect no :=
If it's build time as well, it should be in DEPEND too

Reply:
jgrf is needed only at runtime, the emulation cores do not depend on the frontend. jgrf will load the cores at runtime with `SDL_LoadObject`. It is possible in the future there could be more than one viable frontend.

That said the headers (`jg`), frontend (`jgrf`) and the cores (i.e. `nestopia-jg`) all need to be on the same API version. Now that the project is officially released it is unlikely the API will change in the near future and backward compatibility with a deprecation period is planned in the event it is changed.

I removed the `:=` from the jgrf dependency.

Thanks for the review!